ambry-cloud/src/main/java/com/github/ambry/cloud/azure/ContainerDeletionEntry.java

```diff
@@ -0,0 +1,184 @@
+/**
+ * Copyright 2020 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.cloud.azure;
+
+import com.github.ambry.account.Container;
+import com.github.ambry.clustermap.ClusterMap;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import org.json.JSONObject;
+
+
+/**
+ * Class representing container deletion status in cloud.
+ */
+public class ContainerDeletionEntry {
+
+  static final String VERSION_KEY = "version";
+  static final String CONTAINER_ID_KEY = "containerId";
+  static final String ACCOUNT_ID_KEY = "accountId";
+  static final String CONTAINER_DELETE_TRIGGER_TIME_KEY = "deleteTriggerTime";
+  static final String IS_DELETED_KEY = "isDeleted";
+  static final String DELETE_PENDING_PARTITIONS_KEY = "deletePendingPartitions";
+
+  private static short JSON_VERSION_1 = 1;
+
+  private final short version;
+  private final short containerId;
+  private final short accountId;
+  private final Set<String> deletePendingPartitions;
+  private final long deleteTriggerTimestamp;
+  private boolean isDeleted;
+
+  /**
+   * Constructor for {@link ContainerDeletionEntry}.
+   * @param containerId container id.
+   * @param accountId account id of the container.
+   * @param deleteTriggerTimestamp timestamp at which delete was triggered.
+   */
+  public ContainerDeletionEntry(short containerId, short accountId, long deleteTriggerTimestamp) {
```

Comment:
Where is this method used?  If this will be used in future PR, try to re-use the other ctor to remove duplicate code:
```
  public ContainerDeletionEntry(short containerId, short accountId, long deleteTriggerTimestamp) {
    this(containerId, accountId, deleteTriggerTimestamp, true, Collections.emptyList());
  }
```